torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1791,8 +1791,19 @@ void TensorExprKernel::runWithAllocatedOutputs(Stack& stack) {
     }
   }
 
+  std::vector<int64_t> stride_values(input_stride_args_.size());
   if (has_symbolic_shapes_) {
     updateOutputSizesAndStrides(stack_inputs);
+
+    // add stride args
+    for (auto idx : c10::irange(input_stride_args_.size())) {
+      const auto& input_stride_arg = input_stride_args_[idx];
+      stride_values[idx] =
```

Comment:
we can just use the index of tensor stride right ?
```
args.emplace_back(&stack_inputs[input_stride_arg.first].toTensor().strides().at(
              input_stride_arg.second);
```

Reply:
This requires a const_cast:
```
args.emplace_back(const_cast<long*>(
          &stack_inputs[input_stride_arg.first].toTensor().strides().at(
              input_stride_arg.second)));
```